src/modules/dashboard/components/Whitelist/UploadAddressesWidget/UploadAddressesWidget.tsx

```diff
@@ -35,10 +53,6 @@ const MSG = defineMessages({
     id: `dashboard.Whitelist.UploadAddressesWidget.uploadError`,
     defaultMessage: `We do not accept more than 100 addresses at a time, please upload a smaller amount.`,
   },
-  inputError: {
-    id: `dashboard.Whitelist.UploadAddressesWidget.inputError`,
-    defaultMessage: `TODO`,
-  },
```

Comment:
Is this not used anymore?

Reply:
no, there is a validation error coming from the form so not used any more.